test/ut/tools/nnictl/test_kill_command.py

```diff
@@ -0,0 +1,121 @@
+import argparse
+import multiprocessing
+import os
+import subprocess
+import signal
+import sys
+import signal
+import time
+
+import pytest
+
+from nni.tools.nnictl.command_utils import kill_command, _check_pid_running
```

Comment:
Is this case testing the `kill_command()` function?
I think it's not a public API, and NNI experiments are designed to be stopped with REST API.

Reply:
`kill_command()` means: kill a process and wait for the kill to finish.